en/asgardeo/docs/guides/authentication/mfa/add-duo-login.md

```diff
@@ -0,0 +1,65 @@
+# Add Duo login
+
+[Duo](https://duo.com/) is a multi factor authentication option that offer multiple verification methods such as push notifications, SMS, phone calls, and hardware tokens.
+
+In this guide, you'll learn how to integrate Duo with your Asgardeo application, enabling Duo as a second-factor authentication provider to enhance your application's security.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Duo Account: A Duo administrator account to configure your Duo application.
+- Asgardeo Account: Access to your Asgardeo console with administrative privileges.
+
+## Configure Duo Security Application
```

Comment:
Can't we link to the Duo documentation for this configuring steps.

Reply:
There are no guides provided for this specific flow in the Duo security official documentation. So therefore we may have to provide the step by step guide on our end.